numbat-wasm/www/index.js

```diff
@@ -110,11 +110,31 @@ function setup() {
   });
 }
 
+// TODO: I would prefer to use const lambdas rather than `function` in order to
```

Comment:
I don't plan to merge this comment as-is but to remove it or update this file based on feedback. I'm not confident enough in my JavaScript abilities to use `function` in general unless I'm implementing generators, where it's strictly required. 🙂

Reply:
I offered a suggestion in my review.